colonialsite/members/management/commands/refresh_member_info.py

```diff
@@ -12,37 +14,54 @@
 # todo: add options to only update for certain class years, existing members, etc
 # Command to import member data from TigerBook API
 class Command(BaseCommand):
-	def handle(self, *args, **options):
+    def handle(self, *args, **options):
 
 		try:
 			username = settings.TIGERBOOK_LOGIN
 			password = settings.TIGERBOOK_PW
 		except AttributeError:
 			raise CommandError('Login and password for Tigerbook not set in settings.')
 
-		all_members = Member.objects.all()
+        all_members = Member.objects.all()
 
-		for member in all_members:
-			netid = member.netid
+        # pass 1: remove no-longer-existing members
+        for member in all_members:
+            netid = member.netid
+            if netid not in USER_LIST:
+                # delete the member and user objects
+                member.delete()
+                user_obj = User.objects.get(username=netid)
+                user_obj.delete()
 
-			url = 'https://tigerbook.herokuapp.com/api/v1/undergraduates/' + netid
-			created = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
-			nonce = ''.join([random.choice('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ+/=') for i in range(32)])
+        # pass 2: add Member objects for new members
+        for netid in USER_LIST:
+            try: 
```

Comment:
Can you do this with a filter, like filter out all objects that do not exist, rather than try, raise exception, do work.